recipes/ofono/ofono.inc

```diff
@@ -0,0 +1,58 @@
+DESCRIPTTION = "oFono is a free software project for mobile telephony (GSM/UMTS) applications"
+LICENSE = "GPLv2"
+
+SRC_URI = "https://www.kernel.org/pub/linux/network/ofono/ofono-${PV}.tar.xz"
+
+inherit autotools pkgconfig
+
+DEPENDS += "libdl libdbus-1 libglib libutil"
+DEPENDS += "mobile-broadband-provider-info-dev"
+
+EXTRA_OECONF += "--disable-udev"
+
+RECIPE_FLAGS += "ofono_dundee"
+EXTRA_OECONF += "--disable-dundee"
+EXTRA_OECONF:>USE_ofono_dundee = " --enable-dundee"
+DEFAULT_USE_ofono_dundee = False
+
+RECIPE_FLAGS += "ofono_qmimodem"
+EXTRA_OECONF += "--disable-qmimodem"
+EXTRA_OECONF:>USE_ofono_qmimodem = " --enable-qmimodem"
+DEFAULT_USE_ofono_qmimodem = True
+
+RECIPE_FLAGS += "ofono_threads"
+EXTRA_OECONF += "--disable-threads"
+EXTRA_OECONF:>USE_ofono_threads = " --enable-threads"
+DEPENDS:>USE_ofono_threads = " libgthread"
+DEFAULT_USE_ofono_threads = True
+
+RECIPE_FLAGS += "ofono_test"
+EXTRA_OECONF += "--disable-test"
+EXTRA_OECONF:>USE_ofono_test = " --enable-test"
+DEFAULT_USE_ofono_test = False
+
+RECIPE_FLAGS += "ofono_tools"
+EXTRA_OECONF += "--disable-tools"
+EXTRA_OECONF:>USE_ofono_tools = " --enable-tools"
+DEFAULT_USE_ofono_tools = False
+
```

Comment:
It would be nice if was possible to disable provisioning (mobile-broadband-provider-info).

Maybe like this

RECIPE_FLAGS += "ofono_provision"
EXTRA_OECONF += "${EXTRA_OECONF_PROV}"
EXTRA_OECONF_PROV = "--disable-provision"
EXTRA_OECONF_PROV:USE_ofono_provision = "--with-provisiondb=${datadir}/mobile-broadband-provider-info/serviceproviders.xml"
DEPENDS:>USE_ofono_provision = " mobile-broadband-provider-info"
DEPENDS_${PN}:>USE_ofono_provision = " mobile-broadband-provider-info"
RDEPENDS_${PN}:>USE_ofono_provision = " mobile-broadband-provider-info"
DEFAULT_USE_ofono_provision = True